deps/db/src/logseq/db/frontend/rules.cljc

```diff
@@ -155,64 +155,111 @@
    (dissoc query-dsl-rules :namespace
            :page-property :has-page-property
            :page-tags :all-page-tags)
-   {:tags
+   {:existing-property-value
```

Comment:
I haven't looked at these rules yet but similar to #11565 and #11451 I'm concerned about their complexity . I'm going to look at testing them tomorrow and seeing if there's any simplification we can do. At a minimum I'd like to have a simple *property rules for other features and more complex ones for simple queries

Reply:
Addressed this and added tests for querying default values